### ISSUE TITLE
Optionally disable metric_counters 

### ISSUE THREAD

**Issue by rdzimmer** [kairosdb/kairosdb#466]
In the initial CQL release, the metric_counters were broken:
https://github.com/kairosdb/kairosdb/issues/451
Now that they're back in 1.2.1 they're again adding a small but noticeable overhead on my environment.  The exact amount depends on the tag to metric_name ratio.  In one env I'm at around 10:1, so my datapoint posts went up by ~10%.  I never look at the `kairosdb.metric_counters`.  
However, I do really appreciate a lot of the stats like `kairosdb.http.ingest_count`, so I don't want to turn off the `kairosdb.service.reporter=org.kairosdb.core.reporting.MetricReportingModule` completely or turn down the `kairosdb.reporter.schedule=0 */1 * * * ?`.  Would it be possible to have the option to re-break (disable) the possibly expensive metric_counters?

**Comment by rdzimmer** [comment]
Sounds like this forum topic is related? "A better way to report metrics"
https://groups.google.com/forum/#!topic/kairosdb-group/Yk7yAx1GGPE

**Comment by brianhks** [comment]
I agree, the reporting of metrics needs to be more granular.  I'll add something for this in 1.3

**Comment by brianhks** [comment]
In the short term I've added a filter plugin that lets you filter out certain metrics.  Not the best for this situation but a stop gap until I get Metrics4j Integrated in the next release.
